src/components/layout/Documentation/Card.js

```diff
@@ -0,0 +1,62 @@
+import cx from 'classnames';
+import { GatsbyImage, getImage } from 'gatsby-plugin-image';
+import React from 'react';
+
+import analytics from '../../../analytics';
+import snakecase from '../../../hooks/snakecase';
+import { Link } from '../../default/Link';
+import CardLink from './CardLink';
+
+export default function Card({
+  title,
+  text,
+  image,
+  link,
+  bodyLinkOneURL,
+  bodyLinkOneTitle,
+  bodyLinkTwoURL,
+  bodyLinkTwoTitle,
+  bodyLinkThreeURL,
+  bodyLinkThreeTitle,
+}) {
+  const imageStyles = 'block h-28 sm:h-40 xs:h-40 object-cover rounded-t-md';
+  const cardImage = getImage(image);
+  return (
+    <div
+      className={cx(
+        'bg-substrateGray-light rounded-md shadow-md mb-8',
+        'xl:w-[350px]',
+        'lg:w-[280px] lg:mx-4',
+        'md:mb-10',
+        'sm:mb-20',
+        'xs:w-full',
+        'dark:bg-substrateBlackish',
+        'hover:scale-105 transition-transform'
+      )}
+    >
+      <Link
+        to={link}
+        onClick={() => {
+          analytics.click(snakecase(link));
```

Comment:
could we adapt the previous implementation, remove the `click` method and keep only `track`:

```
click: event => {
    const e = `click_${event}`;
    window.sa_event && window.sa_event(e);
  },
```

and then pass directly 
```
analytics.track(`${click_}snakecase(link))`);
```

Reply:
should be like this no?

` analytics.track(`click_${snakecase(link)}`);`